### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
- [DEMO LINK](https://NikitaKarbazin.github.io/layout_catalog/)
- [TEST REPORT LINK](https://NikitaKarbazin.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,359 @@
...
+            <div class="card__stars-line">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
```
Comment (Kaunin1) [inline src/index.html:81]: blank spasings are not required for single-line elements. Lets remove.

File: src/styles/_card.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 16px;
```
[the remark this answers is not in the document]
Comment (NikitaKarbazin) [reply src/styles/_card.scss]: Yes, thanks it's my carelessness

File: src/index.html (hunk 1 above)
Comment (NikitaKarbazin) [reply src/index.html]: Fixed

**Review by Kaunin1 — APPROVED** [review]

File: src/styles/_card.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $color;
+    line-height: 18px;
+  }
+
+  &__num {
+    color: #616070;
+    font-weight: 400;
```
Comment (illiarerun) [inline src/styles/_card.scss:27]: that's default value
```suggestion
```

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+@import "reset";
+@import "header";
+@import "star";
+@import "card";
+@import "variables";
+
+html {
+  font-family: Roboto, serif;
+
+  .container {
```
Comment (illiarerun) [inline src/styles/main.scss:10]: create separate file for that block

Comment (illiarerun) [inline src/styles/main.scss:10]: why do wrap it with html selector?

File: src/styles/_star.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: inline-flex;
```
Comment (illiarerun) [inline src/styles/_star.scss:2]: why inline-flex?

File: src/styles/_star.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+    &:nth-child(1) {
+      margin-left: 0;
+    }
+
+    .stars--1 .stars__star:nth-child(1),
+    .stars--2 .stars__star:nth-child(-n+2),
+    .stars--3 .stars__star:nth-child(-n+3),
+    .stars--4 .stars__star:nth-child(-n+4),
+    .stars--5 .stars__star:nth-child(-n+5) {
```
Comment (illiarerun) [inline src/styles/_star.scss:21]: you can use loop here

File: src/styles/_header.scss [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  &__logo {
+    margin-left: 50px;
+
+    .header__img {
+      display: flex;
+      align-self: center;
+    }
+  }
+
+  .nav {
```
Comment (illiarerun) [inline src/styles/_header.scss:18]: create separate file for this block

File: src/styles/_header.scss [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    margin-right: 50px;
```
Comment (illiarerun) [inline src/styles/_header.scss:19]: don't add margins to blocks, make it a padding for your header

File: src/styles/main.scss [hunk 9]
```diff
@@ -1,3 +1,28 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    flex-shrink: 0;
+    width: 1024px;
```
Comment (illiarerun) [inline src/styles/main.scss:16]: [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 11th item, use values like card-size, gap and max count of cards in a row to calculate it

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, now let's work on some fixes:

File: src/index.html [hunk 10]
```diff
@@ -4,9 +4,327 @@
...
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <section class="container">
+        <div class="container__item">
```
[the remark this answers is not in the document]
Comment (NikitaKarbazin) [reply src/index.html:63]: Misha explained to us that this is a personal decision for everyone, he has friends who make wrappers and there are those who make mix

File: src/styles/main.scss (hunk 9 above)
Comment (NikitaKarbazin) [reply src/styles/main.scss]: fix

File: src/styles/_header.scss (hunk 8 above)
Comment (NikitaKarbazin) [reply src/styles/_header.scss]: fix

File: src/styles/_header.scss (hunk 7 above)
Comment (NikitaKarbazin) [reply src/styles/_header.scss]: fix

File: src/styles/_star.scss (hunk 6 above)
Comment (NikitaKarbazin) [reply src/styles/_star.scss]: fixed

File: src/styles/_star.scss (hunk 5 above)
Comment (NikitaKarbazin) [reply src/styles/_star.scss]: in this moment the STAR say me that i need to use inline flex

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**